app/src/main/java/org/mozilla/focus/settings/LearnMoreSwitchPreference.kt

```diff
@@ -36,15 +40,10 @@ abstract class LearnMoreSwitchPreference(context: Context?, attrs: AttributeSet?
         learnMoreLink.setOnClickListener {
             // This is a hardcoded link: if we ever end up needing more of these links, we should
             // move the link into an xml parameter, but there's no advantage to making it configurable now.
-            val session = Session(getLearnMoreUrl(), source = Session.Source.MENU)
-            context.components.sessionManager.add(session, selected = true)
-            if (context is ContextThemeWrapper) {
-                if ((context as ContextThemeWrapper).baseContext is Activity) {
-                    ((context as ContextThemeWrapper).baseContext as Activity).finish()
-                }
-            } else {
-                (context as? Activity)?.finish()
-            }
+            val url = getLearnMoreUrl()
+            val intent = InfoActivity.getIntentFor(context!!,
+                    url, context.getString(R.string.enable_search_suggestion_subtitle_learnmore))
```

Comment:
Curious, why do we use the `enable_search_suggestion_subtitle_learnmore` string?

Reply:
Thought 'Learn More' would be a suitable heading. Maybe there is a better option?

Reply:
Nope, was just curious :)